src/Apps/Artist/Routes/AuctionResults/ArtistAuctionResults.tsx

```diff
@@ -274,62 +283,70 @@ const AuctionResultsContainer: React.FC<AuctionResultsProps> = ({
 
           <Spacer y={[2, 0]} />
 
+          {results.length === 0 && (
+            <ArtworkGridEmptyState
+              onClearFilters={() => {
+                resetFilters?.()
```

Comment:
The erroneous nullable typing on all these filter contexts is super frustrating.